### PR TITLE
Added polish language

### PR DESCRIPTION
**Pull request by Radeox**
Create polish translation and added to UI

### CODE REVIEW

File: res/romfs/lang/pl.lang [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+
+Failed to commit title: 0x%08lX
+Finalizacja tytułu nie udała się: 0x%08lX
+
+Failed to finalize TMD: 0x%08lX
+Finalizacja TMD nie udała się: 0x%08lX
+
+Failed to finalize content: 0x%08lX
+Finalizzazione contenuti fallita: 0x%08lX
+Finalizacja zawartość nie udała się: 0x%08lX
```
Comment (Cruel) [inline res/romfs/lang/pl.lang]: Missing a line or something?

Comment (Radeox) [inline res/romfs/lang/pl.lang]: Forgot to delete old one, fixing now

File: src/GUI/Settings.cpp [hunk 2]
```diff
@@ -763,6 +763,7 @@ void Settings::fillOtherPage(Gwen::Controls::Base *page)
 	m_listboxLanguages->AddItem("Português", "pt");
 	m_listboxLanguages->AddItem("Italiano", "it");
 	m_listboxLanguages->AddItem("Nederlands", "nl");
+	m_listboxLanguages->AddItem("Polish", "pl");
```
Comment (Cruel) [inline src/GUI/Settings.cpp]: That is good, but translate "Polish" to the respective language to follow the rest of them.

**Comment by Cruel** [comment]
Thanks!

**Pull request merged**